git_machete/github.py

```diff
@@ -29,20 +28,21 @@
 
 
 class GitHubPullRequest(object):
-    def __init__(self, number: int, user: str, base: str, head: str, html_url: str, state: str) -> None:
+    def __init__(self, number: int, user: str, base: str, head: str, html_url: str, state: str, repo: Dict[str, Any]) -> None:
         self.number = number
         self.user = user
         self.base = base
         self.head = head
         self.html_url = html_url
         self.state = state
+        self.repo = repo
 
     def __repr__(self) -> str:
         return f"PR #{self.number} by {self.user}: {self.head} -> {self.base}"
 
 
 def __parse_pr_json(pr_json: Any) -> GitHubPullRequest:
-    return GitHubPullRequest(int(pr_json['number']), pr_json['user']['login'], pr_json['base']['ref'], pr_json['head']['ref'], pr_json['html_url'], pr_json['state'])
+    return GitHubPullRequest(int(pr_json['number']), pr_json['user']['login'], pr_json['base']['ref'], pr_json['head']['ref'], pr_json['html_url'], pr_json['state'], pr_json['head']['repo'])
```

Comment:
The number of params here is dangerously high... I'd use named params here to mitigate the risk of swapping the params by accident (esp. that all of them are of the same type, so mypy won't report an error)